config/clusters/jupyter-meets-the-earth/cluster.yaml

```diff
@@ -0,0 +1,27 @@
+name: jupyter-meets-the-earth
+provider: aws
+aws:
+  key: enc-deployer-credentials.secret.json
+  clusterType: eks
+  clusterName: jupyter-meets-the-earth
+  region: us-west-2
+support:
+  helm_chart_values_files:
+    - support.values.yaml
+    - enc-support.secret.values.yaml
+hubs:
+  - name: staging
+    domain: staging.hub.jupytearth.org
+    helm_chart: daskhub
+    helm_chart_values_files:
+      - common.values.yaml
+      - staging.values.yaml
+      - enc-staging.secret.values.yaml
+  - name: prod
+    display_name: "Jupyter Meets the Earth"
+    domain: hub.jupytearth.org
```

Comment:
Hmmm should this be jmte.2i2c.cloud?

I got cert issues visiting hub.jupytearth.org

Reply:
Ah, changed, but not sure why the redirects aren't working. Checking that now.

Reply:
@consideRatio good catch, and fixed!